src/VisualStudio/Core/Def/DocumentOutline/DocumentSymbolViewModel.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+using Microsoft.VisualStudio.Imaging;
+using Microsoft.VisualStudio.Imaging.Interop;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.VisualStudio.LanguageServices
+{
+    using SymbolKind = LanguageServer.Protocol.SymbolKind;
+
+    internal class DocumentSymbolViewModel : INotifyPropertyChanged
+    {
+        private bool _isSelected;
+        private bool _isExpanded;
+
+        public string Name { get; }
+
+        public List<DocumentSymbolViewModel> Children { get; set; }
+
+        public int StartLine { get; }
+        public int StartChar { get; }
+        public int EndLine { get; }
+        public int EndChar { get; }
+
+        public SymbolKind SymbolKind { get; }
+        public ImageMoniker ImgMoniker { get; }
```

Comment:
```suggestion
        public ImageMoniker ImageMoniker { get; }
```

Generally speaking, we dont' abbr.